### DISCUSSION TITLE
[Feature Request] Pen input for boox devices

### DISCUSSION THREAD

**Discussion by josephm1** [plateaukao/einkbro#341] [Ideas]
Hey there,

I suggest considering integrating with the ONYX Boox Scribble SDK could enable passing Scribble input from the stylus to HTML canvas elements. This approach offers various benefits, including integration with AI APIs, seamless collaboration with online editing tools and programming suites, and enhanced accessibility options through customizable HTML templates. As someone familiar with web development, I'd be happy to create custom templates if needed.

What do you think about this proposal?

If it was done like this with AI apis you could have a [tom riddle diary like](https://www.youtube.com/watch?v=eh4b5zC0sB4) template as well as much more functional templates.

Additional code for making a webview scribble compatible https://github.com/onyx-intl/OnyxAndroidDemo/blob/master/app/src/main/java/com/android/onyx/demo/scribble/ScribbleWebViewDemoActivity.java

**Comment by braddo99** [comment]
I would love it einkbro used the Boox SDK to allow low latency drawing on HTML5 canvas objects! Then the todo web app I built would become useable. At the moment it's just super laggy. Have tried to build my own Android app with the Onxy SDK and it's challenging - especially since I'm not a developer :-)
